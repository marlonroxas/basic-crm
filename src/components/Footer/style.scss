footer {
  position: fixed;
  bottom: 0;
  width: 100%;
  height: 50px;
  background-color: rgb(250, 250, 250);
  text-align: center;
  line-height: 50px;
  box-shadow: 0 -1px 3px rgba(0, 0, 0, 0.15);
  z-index: 10;
  .copyright {
    color: #555;
  }
}
