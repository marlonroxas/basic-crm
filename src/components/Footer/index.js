import React from "react";

// Styles
import "./style.scss";

const Footer = () => {
  return (
    <footer className="footer">
      <div className="container">
        <small className="copyright">Basic CRM by Marlon Roxas</small>
      </div>
    </footer>
  );
};

export default Footer;
