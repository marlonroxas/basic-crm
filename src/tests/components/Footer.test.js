import React from 'react'
import { shallow } from 'enzyme'

import Footer from '../../components/Footer/index'

test('Should render Footer correctly', () => {
  const wrapper = shallow(<Footer />)
  expect(wrapper).toMatchSnapshot()
})