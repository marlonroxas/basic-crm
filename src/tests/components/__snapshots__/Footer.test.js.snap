// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Footer correctly 1`] = `
<footer
  className="footer"
>
  <div
    className="container"
  >
    <small
      className="copyright"
    >
      Basic CRM by Marlon Roxas
    </small>
  </div>
</footer>
`;
